server/sonar-server/src/main/java/org/sonar/server/component/ws/SearchProjectsAction.java

```diff
@@ -57,11 +57,15 @@
   private final DbClient dbClient;
   private final ProjectMeasuresIndex index;
   private final ProjectMeasuresQueryValidator projectMeasuresQueryValidator;
+  private final ProjectMeasuresQueryFactory queryFactory;
+  private final UserSession userSession;
 
-  public SearchProjectsAction(DbClient dbClient, ProjectMeasuresIndex index, ProjectMeasuresQueryValidator projectMeasuresQueryValidator) {
+  public SearchProjectsAction(DbClient dbClient, ProjectMeasuresIndex index, ProjectMeasuresQueryFactory queryFactory, ProjectMeasuresQueryValidator projectMeasuresQueryValidator, UserSession userSession) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 206 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00103)